pkg/ipam/allocator.go

```diff
@@ -44,7 +44,7 @@ var (
 )
 
 // AllocateIP allocates a IP address.
-func (ipam *IPAM) AllocateIP(ip net.IP, owner string) (err error) {
+func (ipam *IPAM) AllocateIP(ip net.IP, owner string, refresh bool) (err error) {
```

Comment:
Please introduce a new function `AllocateIPAndSyncUpstream()` instead of adding an argument. It will document the difference between refreshing the node immediately or postponing it. Reading `true` or `false` to a function argument makes the code hard to understand without looking up the definition of the function.